web-test-runner.utils.js

```diff
@@ -134,24 +134,24 @@ const vrtHTML =
     </html>`;
 
 export let vrtGroups = [];
-const themeVariants = ['classic', 'express'];
+const systemVariants = ['spectrum', 'express'];
```

Comment:
should we add 'spectrum-two' here too?

Reply:
Not yet, as it's not a real theme yet. So I left it off.